libs/ledger-live-common/src/families/polkadot/api/sidecar.ts

```diff
@@ -104,15 +118,28 @@ const fetchStashAddr = async (addr: string): Promise<string | null> => {
  * @returns {string}
  */
 const fetchControllerAddr = async (addr: string): Promise<string | null> => {
-  const {
-    data,
-  }: {
-    data: SidecarPalletStorageItem;
-  } = await network({
-    method: "GET",
-    url: getSidecarUrl(`/pallets/staking/storage/bonded?key1=${addr}`),
```

Comment:
Have you tried just providing both path params in the same query?
i.e:
```
getSidecarUrl(`/pallets/staking/storage/bonded?key1=${addr}&keys[]=${addr}`),
```

Reply:
Nice I didn't think about this let me try

Reply:
Seems to work perfect !  I really do like this solution Thanks